vocabularies/trs.ttl

```diff
@@ -235,36 +236,38 @@ trs:UTCTime
   skos:prefLabel "Coordinated Universal Time"@en ;
   skos:topConceptOf <http://linked.data.gov.au/def/trs> ;
 .
-
 trs:ThousandsOfYearsAgo
   a skos:Concept ;
   a time:TRS ;
-  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
-  skos:altLabel "ka"@en ;
   skos:broader trs:BeforePresent ;
   skos:inScheme <http://linked.data.gov.au/def/trs> ;
   skos:definition "Temporal position expressed numerically scaled in thousands of years increasing backwards relative to 1950. Calibration to 1950 not always significant or done, except when carbon dating methods are used."@en ;
   skos:prefLabel "Thousands of Years Ago"@en ;
 .
-
 trs:MillionsOfYearsAgo
   a skos:Concept ;
   a time:TRS ;
-  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
   skos:altLabel "Ma"@en ;
   skos:broader trs:BeforePresent ;
   skos:inScheme <http://linked.data.gov.au/def/trs> ;
   skos:definition "Temporal position expressed numerically scaled in millions of years increasing backwards relative to 1950. Calibration to 1950 usually not significant or done."@en ;
   skos:prefLabel "Millions of Years Ago"@en ;
 .
-
 trs:BillionsOfYearsAgo
   a skos:Concept ;
   a time:TRS ;
-  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
   skos:altLabel "Ga"@en ;
   skos:broader trs:BeforePresent ;
   skos:inScheme <http://linked.data.gov.au/def/trs> ;
   skos:definition "Temporal position expressed numerically scaled in billions of years increasing backwards relative to 1950. Calibration to 1950 usually not significant or performed."@en ;
   skos:prefLabel "Billions of Years Ago"@en ;
 .
+trs:BeforePresentCalibrated
+  a skos:Concept ;
+  a time:TRS ;
+  skos:altLabel "BP-Cal"@en ;
+  skos:broader trs:BeforePresentCalibrated ;
+  skos:inScheme <http://linked.data.gov.au/def/trs> ;
+  skos:definition "A time scale used mainly in archaeology, geology and other scientific disciplines to specify when events occurred in the past. Usually used when carbon dating was the method used to determine the age. Because the 'present' time changes, standard practice is to use 1 January 1950 as the commencement date (epoch) of the age scale. A calibrated mean the age has been adjusted to take into account the variation of the carbon 14 to carbon 12 ratio in the atmosphere through time."@en ;
```

Comment:
Because the parent Before Present concept is already defined, and to make the specificity more clear I would suggest altering the definition structure slightly. Something like 

> skos:definition "A calibrated Before Present age that has been adjusted to correct for the variation of the carbon 14 to carbon 12 ratio in the atmosphere through time. A Before Present age is a time scale used to specify when events occurred in the past. It is primarily used when carbon dating the method of age determination. "@en

Reply:
I think for the geological and archeological communities that are the people who are going to be using this timescale the most, the category under which these timescales would be better called "Geologic and Archeologic Timescales", but I don't understand your naming conventions and procedures. If the category needs to remain "Before Present" then changing to this definition makes sense.